### DISCUSSION TITLE
Missing analytics on extension dashboard

### DISCUSSION THREAD

**Discussion by Trafikant** [microsoft/MicrosoftEdge-Extensions#194] [Discussions]
I have an extension with more than 5000 WAU, but still, there's no analytics data in the Analytics tab. It's been like this since day one. Is there anything needs to e doe on my side or is it an Edge bug?

![image](https://github.com/user-attachments/assets/11e4fc3a-72db-4a61-9daa-30609219c7fc)

Extension link - 
https://microsoftedge.microsoft.com/addons/detail/taghound-analyticsgtm/kgfpeplfihidbbbnmhpknhpkikbijoci

**Comment by ManikanthMSFT** [comment]
Hi @Trafikant, thank you for reporting this issue. We are currently looking into it and will update you as soon as we have more information.

**Comment by Trafikant** [comment]
Hi, any findings?

**Reply by ManikanthMSFT** [reply]
Hi @Trafikant, apologize for the inconvenience and understand your frustration. Our team is actively working on this issue. We will provide you with an update as soon as we have more information.

**Comment by ManikanthMSFT** [comment]
Hey @Trafikant, thank you for your patience while we worked to resolve the issue you were experiencing. I'm pleased to inform you that our engineering team has identified and fixed the problem. The data should flow correctly within the next 24 hours.

Please feel free to reach out if you notice any further issues or have any additional questions. We appreciate your understanding and look forward to ensuring everything runs smoothly for you.
